app/src/main/java/org/p2p/wallet/svl/model/SendLinkGenerator.kt

```diff
@@ -59,6 +57,11 @@ object SendLinkGenerator {
         )
     }
 
+    fun isValidSeedCode(symbols: List<String>): Boolean {
+        val regex = Regex(REGEX_LINK_ALLOWED_SYMBOLS)
+        return symbols.all { regex.matches(it) }
```

Comment:
add check for SYMBOLS_COUNT too?

Reply:
good one! 
I was checking for count in interactor. 
Moved here